index.html

```diff
@@ -191,27 +193,21 @@ <h2 class="section-header">About Myself</h2>
       </section>
 
       <section class="contact" id="contact">
-        <div class="contact-wrapper">
-          <h3 class="section-header">Contact Me</h3>
-          <h5>
-            If you have an application you are interested in developing,
-            a feature that you need built or a project that needs coding.
-            I’d love to help with it.
-          </h5>
-          <form action="#" class="contact-form">
-            <input type="text" name="name" id="name" placeholder="Name">
-            <input type="email" name="email" id="email" placeholder="E-mail">
-            <textarea 
-            name="message" 
-            id="message" 
-            cols="30" 
-            rows="5" 
-            placeholder="Write your message here"></textarea>
-            <button type="submit" class="buttons">Get In Touch</button>
+        <div class="contact_wrapper">
+          <h2 class="section-header">Contact me</h2>
+          <p class="description">If you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          <form action="#" method="post">
+            <input type="text" name="user_name" placeholder="Yeremias" required max="30">
+            <input type="email" name="user_mail" placeholder="name@gmail.com" required>
+            <textarea name="user_message" cols="30" rows="5" placeholder="Write your message here" required
+              maxlength="500"></textarea>
+            <button class="buttons" type="submit">Get in Touch</button>
```

Comment:
- [ ] Please Implement `button` interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template. 
![image](https://user-images.githubusercontent.com/97521557/206009777-dae1347c-35ac-4f8c-a808-5e22a1c5a6aa.png)